src/NServiceBus.Gateway.AcceptanceTests/NServiceBus.Gateway.AcceptanceTests.csproj

```diff
@@ -1,16 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net452</TargetFramework>
-    <DebugSymbols>True</DebugSymbols>
+    <TargetFrameworks>net452;netcoreapp2.0</TargetFrameworks>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <NoWarn>PCR0001</NoWarn>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
   </PropertyGroup>
   <ItemGroup>
 
     <ProjectReference Include="..\NServiceBus.Gateway\NServiceBus.Gateway.csproj" />
-    <Reference Include="System.Configuration" />
     <Reference Include="System.Web" />
+    <Reference Include="System.Configuration" />
```

Comment:
These references should be scoped to net452 only. I suppose they might just be ignored for netcoreapp2.0, but even if they are, I'd prefer to see them properly scoped.